lib/Common/Common/NumberUtilities.inl

```diff
@@ -72,7 +72,7 @@ namespace Js
     NUMBER_UTIL_INLINE INT64 NumberUtilities::TryToInt64(double T1)
     {
         INT64 T4_64;
-#if defined(_M_IX86)
+#if defined(_M_IX86) && defined(_WIN32) // TODO: xplat support
```

Comment:
Why not do the same thing you did for AutoSystemInfo::Data::SSE2Available, and just have that function return false with a todo